### PR TITLE
Prepare ovn2sdn automation

### PR DESCRIPTION
**Pull request by weliang1**
Update existing steps to prepare ovn2sdn automation

@openshift/team-sdn-qe PTAL

OVN migrate to SDN
oc get network.operator cluster  -o jsonpath={.spec.migration.networkType}
return: OpenShiftSDN

SDN migrate to OVN
oc get network.operator cluster  -o jsonpath={.spec.migration.networkType}
return: OVNKubernetes

### CODE REVIEW

File: features/step_definitions/networking.rb [hunk 1]
```diff
@@ -1633,3 +1633,17 @@
     end
   end
 end
+
+Given /^the cluster is not migration from ovn plugin$/ do
+  ensure_admin_tagged
+  _admin = admin
+  if env.version_le("4.21", user: user)
```
Comment (liangxia) [inline features/step_definitions/networking.rb:1640]: Is the version `4.21` expected ?

Comment (weliang1) [reply features/step_definitions/networking.rb]: > Is the version `4.21` expected ?

Update it to be 4.12

**Comment by anuragthehatter** [comment]
Thanks Weibin.

/lgtm

**Pull request merged**